vector/src/gplay/AndroidManifest.xml

```diff
@@ -9,17 +9,12 @@
             android:name="firebase_analytics_collection_deactivated"
             android:value="true" />
 
-        <receiver
-            android:enabled="true"
-            android:name=".push.fcm.EmbeddedFCMDistributor"
-            android:exported="false">
-
+        <service android:name="im.vector.app.push.fcm.VectorFirebaseMessagingService"
```

Comment:
@appndigital why don't you keep the change about tools : "<!-- Add tools:ignore="Instantiatable" for the error reported only by the CI :/ -->"